docs/content/direct/common-setup-hub-chart.md

```diff
@@ -0,0 +1,87 @@
+# Common Setup - hub chart
+
+For more details please see [here](./hub-chart.md).
+
+This documents explains how to use KubeStellar Hub chart to deploy a new instance of KubeStellar
+with a choice of user-defined KubeFlex Control Planes (CPs).
+
+The information provided is specific for the following release:
+
+```shell
+export KUBESTELLAR_VERSION=0.23.0-alpha.2
+```
+
+It may also be a good idea to do a bit of cleanup first. See how it is done in the cleanup script for our E2E tests (in `test/e2e/common/cleanup.sh`).
+
+## Create a Kind cluster
+
+The setup of KubStellar via the Hub chart requires the existance of at least one cluster
+to be used for the deployment of the chart.
+
+For convenience, a new local Kind cluster that satisfies the requirements for KubeStellar setup
+and that can be used to exercises the [examples](./examples.md) can be created with the following command:
+
+```shell
+sudo su <<EOF
+bash <(curl -s https://raw.githubusercontent.com/kubestellar/kubestellar/v${KUBESTELLAR_VERSION}/scripts/create-kind-cluster-with-SSL-passthrough.sh) --name kubeflex --port 9443

```

Comment:
FYI, I do not expect the create-kind-cluster-with-SSL-passthrough.sh script to change with every release. I think that this line could have a literal `v0.23.0` rather than `v${KUBESTELLAR_VERSION}` (once v0.23.0 exists). If we ever do change it in a later release then the literal release number can be changed then. Not an issue for this PR, since release 0.23.0 does not exist yet.